.gitignore

```diff
@@ -4,4 +4,5 @@ packages/collector/test/bin/edgemicroResources/cli/edgemicro
 nodejs-license-report.csv
 test-duration-breakdown
 .nyc_output
-coverage
\ No newline at end of file
+coverage
+db2
```

Comment:
I added db2 volume in the docker compose file.
This creates a folder in our root dir. Might be helpful for debugging later